genai-perf/genai_perf/parser.py

```diff
@@ -199,9 +198,10 @@ def _check_conditional_args(
         if endpoint_config.endpoint:
             args.endpoint = endpoint_config.endpoint.format(MODEL_NAME=model_name)
 
-    if args.service_kind == "triton" and args.endpoint_type == "kserve":
+    if args.service_kind == "triton" and args.endpoint_type in ["kserve", "template"]:
```

Comment:
Fix for this endpoint-type, since this converts the string to be an Enum, which is important to avoid type errors later. Long-term, we'll move away from service-kind.